webui/src/lib/api/index.js

```diff
@@ -749,8 +768,9 @@ class Objects {
     async upload(repoId, branchId, path, fileObject, onProgressFn = null) {
         const query = qs({path});
         const uploadUrl = `${API_ENDPOINT}/repositories/${encodeURIComponent(repoId)}/branches/${encodeURIComponent(branchId)}/objects?` + query;
-        const {status, body, contentType} = await uploadWithProgress(uploadUrl, fileObject, 'POST', onProgressFn)
+        const {status, body, rawHeaders} = await uploadWithProgress(uploadUrl, fileObject, 'POST', onProgressFn)
         if (status !== 201) {
+			const contentType = rawHeaders ? parseRawHeaders(rawHeaders)['content-type'] : undefined;
```

Comment:
This func is being used when lakefs is configured not to use presigned url. The change I introduced is due to the change in uploadWithProgress that only returns raw headers. Tested this manually, and in case of an error the func does extract the header and the error message.

Reply:
Great.
Only the indentation again 😅